zipline/data/minute_bars.py

```diff
@@ -465,6 +489,17 @@ def __init__(self,
     def first_trading_day(self):
         return self._start_session
 
+    def ohlc_ratio_for_sid(self, sid):
+        if self._ohlc_ratios_per_sid is not None:
+            try:
+                return self._ohlc_ratios_per_sid[sid]
+            except KeyError:
```

Comment:
Never mind my comment, the `pass` does fall through correctly. It did read slightly off to me at first, but I believe passing on the KeyError is the correct way to go about this.